src/components/coffeechat.ts

```diff
@@ -0,0 +1,273 @@
+import { Client } from 'discord.js-commando';
+import { Database } from 'sqlite';
+import { openDB } from './db';
+import _ from 'lodash';
+import { Person, stableMarriage } from 'stable-marriage';
+
+const COFFEE_ROLE_ID: string = process.env.COFFEE_ROLE_ID || '.';
+const TARGET_GUILD_ID: string = process.env.TARGET_GUILD_ID || '.';
```

Comment:
don't forget to include these in all the `.env` files. An alternative would be to create config files in the `config` directory, and specify these values for each environment. For example, in `config/production/vars.json`, add the variables for the production environment. Since role ID and guild ID are not sensitive info, I think it'd be ok for them to be part of the code.